src/questions.js

```diff
@@ -0,0 +1,156 @@
+const starterQuestions = [
+  {
+    type: "input",
+    name: "title",
+    message: "What is the title of your project?",
+  },
+  {
+    type: "input",
+    name: "description",
+    message: "Provide a description for your Project",
+  },
+  {
+    type: "checkbox",
+    name: "requiredProcesses",
+    message:
+      "Check the box if your project requires any of the following procedures",
+    choices: [
+      { name: "Installation", value: "installation" },
+      { name: "Usage", value: "usage" },
+      { name: "Tests", value: "tests" },
+      { name: "None of these procedures are required", value: "none" },
+    ],
+  },
+];
+
+const processTypeQuestions = (userCheckedProcesses) => {
+  if (!userCheckedProcesses.includes("none")) {
+    return userCheckedProcesses.map((each) => {
+      const obj = {
+        type: "input",
+        name: `${each}Process`,
+        message: `What is the projects (first) ${each} process step?`,
+      };
+
+      return obj;
+    });
+  }
+};
+
+const installationNextStep = (userCheckedProcesses) => {
```

Comment:
Again here you are using a built in inquirer method, but is it really necessary? We have a `while` loop we can make use of in vanilla javascript, which would be a lot simpler to write and also understand as a second developer reading your code.